main.go

```diff
@@ -0,0 +1,61 @@
+package main
```

Comment:
The convention is to have the command main file under `cmd/` package

Reply:
I'm noticing a lot of people put it under a folder in `cmd/`, should I do that too or just have `cmd/main.go`?
![image](https://github.com/devguyio/kuby/assets/99439005/e000e64d-6272-419d-ba30-64f7d2ef5547)

Reply:
it depends on the application, when multiple binaries are built out of the the same repo then it's typical indeed to create a separate directory under cmd. But in this case, it's only a single binary so I'm fine with `cmd/main.go`